HabitTracker.ngalantino/HabitTracker.ngalantino/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "HabitTracker.ngalantino": {
+      "commandName": "Project",
+      "workingDirectory": "C:\\Users\\Shadow\\source\\CodeReviews.Console.HabitTracker\\HabitTracker.ngalantino\\HabitTracker.ngalantino\\"
```

Comment:
Just an FYI that I had to change this value for my machine in order for it to run.
Make sure that if you include hardcoded paths like this that you document it in the README file under an Installation or Usage section.